specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json

```diff
@@ -187,6 +182,10 @@
           "description": "The minimum version required to install.",
           "type": "string"
         },
+        "minOemVersionRequired": {
```

Comment:
add new property  is breaking change, suggest to do it in new service tag

Reply:
Why is adding a property considered a breaking change? Also, the resource provider has always returned this property in the JSON properties of the response object, we are just updating the spec to catch up with the implementation.

Reply:
here' s definition of breaking change https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?rtime=J7qPU_oc10g&CID=CC853967-47A0-4658-B36C-05C499E24BBF&wdLOR=c283536CE-B533-477C-A684-DECA5F071DD3
```
If a new property/field is added to the response an API, the GET-PUT pipeline will be broken. Consider the case where from portal a customer updates the value of a new property "A". Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.  
```

In this case, if the fixing is to fix swagger file, it's ok.